web/js/components/sidebar/charting-mode-options.js

```diff
@@ -297,8 +297,9 @@ function ChartingModeOptions(props) {
     try {
       const response = await fetch(simpleStatsURI, requestOptions);
       const data = await response.text();
+      const parsedData = JSON.parse(data);
       // This is the response when the imageStat server fails
-      if (data === 'Internal Server Error') {
+      if (data === 'Internal Server Error' || data === null || parsedData.status === 204) {
         return {
           ok: false,
```

Comment:
We should account for the various "no data" modes.

Internal Server Error from the API is fine using the same error message (line373).
data == null & status = 204 indicate the request was unable to find data but is not an API error state. Telling the users to try again in a few minutes is not good advise as they will just get the same response. Can we pop up a different message for these cases? Something like "No data was found for this request. Please check the layer, date(s) & location to process & try again."